docs/user/dashboard/dashboard.asciidoc

```diff
@@ -228,6 +228,13 @@ To view the data field summary information, navigate to the field, then click *i
 [role="screenshot"]
 image::images/lens_data_info.png[Data summary window]
 
+*Lens* inspects the top 10 values to build the value distribution: every other value, together with documents with no data for the inspected field, are shown with the "Other" entry in the distribution chart.
```

Comment:
This sentence is missing information about what happens for number/date fields: in both cases we do something more than just top 10. For numbers we show both top 10 and histogram. We show the histogram when the top 10 values are less than 10% of all the values.